index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import CreateCard from './createcard.js'
+
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      view: 'create',
+      flashcards: []
+    }
+    this.addCard = this.addCard.bind(this)
+  }
+
+  addCard(topic, sideA, sideB) {
```

Comment:
Have `addCard` take an `Object` that's ready to go, instead of a list of parameters. Only call `addCard` from your form when it's complete.